src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -52,7 +52,7 @@ public partial class DynamoModel : INotifyPropertyChanged, IDisposable, IEngineC
         private readonly PathManager pathManager;
         private WorkspaceModel currentWorkspace;
         private Timer backupFilesTimer;
-
+        private Dictionary<Guid, string> backupFilesDict = new Dictionary<Guid, string>();
```

Comment:
Sure that sounds fine too, I was imagining making `PreferenceSettings.BackupFiles` a `Dictionary` will involve more work to serialize/deserialize it, so this is okay for now.